### DISCUSSION TITLE
Decimal point cannot be inputted in the Chinese input method environment

### DISCUSSION THREAD

**Discussion by tangtingtt** [dm4t2/vue-currency-input#411] [Q&A] [answered]
Hello! I would like to ask a question: 
If the computer input method is switched to Chinese, enter 10.1, delete. 0, and the decimal point cannot be entered again. If you re-enter another number and enter the decimal point again, you can successfully input it. Why is this? This issue will not occur in an English input method environment. 
 I have reproduced this issue in Chrome, Edge, and Firefox browsers.

**Comment by dm4t2** [comment, marked as answer]
Unfortunately, I'm not familiar with the Chinese input method environment and can therefore not reproduce this.

**Reply by tangtingtt** [reply]
Hello, thank you very much for your reply. I think I have found the reason why decimal points cannot be entered in the Chinese environment, but I cannot change it. Is there a way for me to redefine the DECIMAL_SEPARATORS variable of the currencyFormat.ts file in your code? I just need to add a Chinese dot in it，'。', Just like: const DECIMAL_ SEPARATORS=['。',',', '.', '٫'];, At present, I have not found a way to modify your variable. Thank you!
![1](https://github.com/dm4t2/vue-currency-input/assets/26892355/7e23a9de-eb9d-4c4b-98b9-e459f2b16136)

**Reply by dm4t2** [reply]
I will add the Chinese dot to the supported decimal separators 👍

**Reply by tangtingtt** [reply]
Okay, thank you for your reply. Looking forward to your update. Happy every day.

**Reply by dm4t2** [reply]
Now available in [v3.1.0](https://github.com/dm4t2/vue-currency-input/releases/tag/3.1.0) 🎉
